database/kv/zk/client.go

```diff
@@ -292,6 +288,7 @@ func (z *ZookeeperClient) UnregisterEvent(zkPath string, event *chan struct{}) {
 	for i, e := range infoList {
 		if e == event {
 			infoList = append(infoList[:i], infoList[i+1:]...)
+			close(event)
```

Comment:
Maybe panic because close the event many times when Register multi-path with one event. It's Implicit.

Reply:
Thank u, I'll fix it.